src/main/java/org/broadinstitute/hellbender/engine/datasources/ReferenceWindowFunctions.java

```diff
@@ -12,9 +12,20 @@
 public class ReferenceWindowFunctions {
 
     /**
-     * A function for requesting only reference bases that directly overlap each read
+     * A function for requesting only reference bases that directly overlap each read. To maintain the ability to serialize the reference,
+     * this needs to be a class rather than a simple lambda function because of errors in serializing lambda functions
+     * in Kryo, at least in version 2.21 which we currently using. Partial support for lambda serialization was added in Kryo 3.0.
```

Comment:
Can you create a ticket reminding us to move to Kryo 3.0 when possible so that we can take advantage of the new lambda serialization.